app/controllers/my-services/get-index-controller.js

```diff
@@ -8,53 +8,61 @@ const { response } = require('../../utils/response')
 const serviceService = require('../../services/service_service')
 const getHeldPermissions = require('../../utils/get_held_permissions')
 
-module.exports = (req, res) => {
-  const servicesRoles = _.get(req, 'user.serviceRoles', [])
-  const newServiceId = _.get(req, 'query.s')
+const hasAccountWithPayouts = function hasLiveStripeAccount (gatewayAccounts) {
```

Comment:
A small unit test to cover the happy path for this might be good, we have fixtures for services + gateway accounts so hopefully should be just writing one or two scenarios